styles.css

```diff
@@ -82,3 +82,109 @@ ul {
   align-items: center;
   margin: auto;
 }
+
+/* css for the second part of the project */
+
+.works {
+  align-items: center;
+  text-align: center;
+}
+
+.recent-works {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.img-wrapper {
+  height: 474px;
+  width: 327px;
```

Comment:
- [ ] I noticed that you applied a width of 327px to your `img-wrapper` and that is why you're having issues with the image. If you remove it, it should solve that issue. :100: